tests/integration/informer_mock_test.go

```diff
@@ -0,0 +1,29 @@
+package biz
+
+import (
+	"context"
+	"github.com/docker/docker/api/types"
+)
+
+const (
+	containerID = "my-container"
+	pid         = 666
+)
+
+type InspectorMock struct{}
+
+func (i InspectorMock) ContainerInspect(_ context.Context, _ string) (types.ContainerJSON, error) {
+	return types.ContainerJSON{
+		ContainerJSONBase: &types.ContainerJSONBase{
+			ID: containerID,
+			State: &types.ContainerState{
+				Pid: pid,
+			},
+			RestartCount: 2,
```

Comment:
I would set `containerID`, `pid` and `RestartCount` values as `InspectorMock` attributes, I think the mock would be more flexible.

Reply:
done